packages/core/src/context/MemoryStore.ts

```
@@ -0,0 +1,443 @@
+      indexedMemories: 0,
+      status: this.indexStatus,
+      error: this.lastError,
+    };
+
+    try {
+      // Count total memories
+      const memories = await this.readAllMemories();
+      stats.totalMemories = memories.length;
+
+      // Get indexed count from vector service
+      if (this.vectorService && this.vectorService.isAvailable()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)